zerver/lib/bugdown/__init__.py

```diff
@@ -940,6 +949,8 @@ def extendMarkdown(self, md, md_globals):
             '_begin')
         md.inlinePatterns.add('usermention', UserMentionPattern(mention.find_mentions), '>backtick')
         md.inlinePatterns.add('emoji', Emoji(r'(?<!\w)(?P<syntax>:[^:\s]+:)(?!\w)'), '_end')
+        md.inlinePatterns.add('unicodeemoji', UnicodeEmoji(ur'(?<!\w)(?P<syntax>[\U0001F300-\U0001F64F\U0001F680-\U0001F6FF\u2600-\u26FF\u2700-\u27BF])(?!\w)'), '_end')
```

Comment:
this is already merged, but we should fix that; not sure whether it's worth adding a check for these in our py3k linter.